images-list

```diff
@@ -330,5 +332,6 @@ sonobuoy/sonobuoy rancher/mirrored-sonobuoy-sonobuoy v0.16.3
 sonobuoy/sonobuoy rancher/sonobuoy-sonobuoy v0.19.0
 squareup/ghostunnel rancher/mirrored-squareup-ghostunnel v1.5.2
 squareup/ghostunnel rancher/squareup-ghostunnel v1.5.2
+thanosio/thanos rancher/mirrored-thanosio-thanos v0.15.0
```

Comment:
Why are we using thanos? Monitoring V1 doesn't have support for it

Reply:
https://github.com/rancher/system-charts/blob/4960259f853d6dc10fe8fd346cd38423daf42626/charts/rancher-monitoring/v0.2.2/charts/prometheus/templates/prometheus.yaml#L98-L103 apparently we do support Thanos in Monitoring V1, although it's probably only an option configurable as part of the overrides in Monitoring V1 today.